clients/unshaded/src/main/java/tachyon/client/file/options/CreateFileOptions.java

```diff
@@ -178,9 +178,8 @@ public String toString() {
   /**
    * @return Thrift representation of the options
    */
-  // TODO(calvin): Rename CreateTOptions to CreateFileTOptions
-  public CreateTOptions toThrift() {
-    CreateTOptions options = new CreateTOptions();
+  public CreateFileTOptions toThrift() {
```

Comment:
What do you think of adding a `FileSystemOptions` interface that has the `toThrift` method?